servlet/src/main/java/io/grpc/servlet/AsyncServletOutputStreamWriter.java

```diff
@@ -88,8 +87,8 @@ final class AsyncServletOutputStreamWriter {
     Logger logger = Logger.getLogger(AsyncServletOutputStreamWriter.class.getName());
     this.log = new Log() {
       @Override
-      public boolean isLoggable(Level level) {
-        return logger.isLoggable(level);
```

Comment:
I think the point was to not run this line. There can be synchronization involved here, and that could maybe mess up lincheck. So it wasn't about avoiding the loading of the dependency (that actually happens anyway, even with this change), it ways about not _calling_ into its infrastructure.

Not that I have any more background on this than you do, but I do think it is a plausible story.

Maybe we should change the comment to say "not call java.util.logging.Logger"?